silverstripe/framework/CVE-2020-26136.yaml

```diff
@@ -0,0 +1,11 @@
+title:     "CVE-2020-26136 GraphQL doesn't honour MFA when using basic auth"
+link:      https://www.silverstripe.org/download/security-releases/cve-2020-26136
+cve:       CVE-2020-26136
+branches:
+    3.0.x:
+        time:     2021-06-07 22:31:00
+        versions: ['>=3.0.0', '<3.5.0']
+    4.0.x:
+        time:     2021-06-08 01:46:00
+        versions: ['>=4.0.0-alpha1', '<4.0.0-alpha2']
+reference: composer://silverstripe/graphql
```

Comment:
Please create a separate "silverstripe/graphql" directory for the graphql package, the silverstripe/framework directory is only meant for the silverstripe/framework package.